scripts/v6/asset-admin.sql

```diff
@@ -0,0 +1,115 @@
+-- Various tables used to maintain metadata, or manage components within th eWIP or AR.
```

Comment:
This should be split up:
- refcounter,finyear can stay here
-  changeiconfin is part of the asset register (it is the asset register change log)  
-  I think project should be in a file called v6Wip or something similar.

Please split schema and population. As I have previously mentioned before the real value is in sharing the schema - the data is likely to change depending on the test being run.